app/src/main/res/layout/add_transaction_fragment.xml

```diff
@@ -27,21 +27,21 @@
             android:layout_height="wrap_content"
             android:ems="10"
             android:inputType="textPersonName"
-            android:text="@string/AmountTxt" />
+            android:text="@string/text_amount" />
 
         <TextView
             android:id="@+id/lblCategory"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
-            android:text="Category" />
+            android:text="@string/text_category" />
```

Comment:
Changed to proper string format

Reply:
Externalizing to strings.xml is a good idea.